### PR TITLE
Students can see items needing grading in Todo if they have TA roles in other courses

### PR DESCRIPTION
**Pull request by idevainstr**
refs: MBL-15322
affects: Student
release note: Hide TA role todos.
test plan: see ticket

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 92.03% | 92.03% | 0%

[MBL-15322](https://instructure.atlassian.net/browse/MBL-15322)

#### Affected Apps: Student

#### Release Note: 
Hide TA role todos.
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Todos/Todo.swift [hunk 1]
```diff
@@ -99,8 +99,13 @@ class GetTodos: CollectionUseCase {
 
     var cacheKey: String? { nil }
     var request: GetTodosRequest { GetTodosRequest() }
-
-    var scope: Scope { Scope(predicate: .all, order: [
+    var todoPredicate: NSPredicate {
+        guard AppEnvironment.shared.app == .student else {
```
[the remark this answers is not in the document]
Comment (idevainstr) [reply Core/Core/Todos/Todo.swift]: done.

**Review by vargaat — APPROVED** [review]
QA+1

**Review by balintbartok — APPROVED** [review]
QA + 1

**Pull request merged**